src/test-run/session-controller.js

```diff
@@ -6,6 +6,16 @@ import TestRun from './';
 const ACTIVE_SESSIONS_MAP = {};
 const UPLOADS_DIR_NAME = '_uploads_';
 
+class CookieMock {
```

Comment:
Let's add a comment and use the `stub` word in naming (we usually use the `mock` in tests).

```js
// NOTE: Proxyless cookie implementation doesn't require client-server communication. This stub was created to reduce conditional logic in connected classes.
class ProxylessCookieStub {
```

Reply:
I've deleted this yet for current proxyless implementation. I'll apply these changes for the next iteration